python/tk_multi_publish2/dialog.py

```diff
@@ -349,10 +348,10 @@ def _create_item_details(self, tree_item):
         
         # if summary thumbnail is defined, item thumbnail should inherit it
         # unless item thumbnail was set after summary thumbnail
-        if self._summary_thumbnail != None and not item._thumbnail_overrides_summary:
+        if self._summary_thumbnail != None and not item.thumbnail_explicit:
            item.thumbnail = self._summary_thumbnail
            # activate snaphot on inherited thumbnail, that way we can override it for a specific item
-           item._set_thumbnail_enabled(True)
+           self.ui.item_thumbnail.setEnabled(True)
```

Comment:
I think we should take this line out. Like I mentioned, some items are explicitly disabled from setting a thumbnail. jpeg items for example. See lines `329-339`.